presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -319,69 +324,69 @@ Configuration Properties
 
 The following configuration properties are available for all catalog types:
 
-======================================================= ============================================================= ============
-Property Name                                           Description                                                   Default
-======================================================= ============================================================= ============
-``iceberg.catalog.type``                                The catalog type for Iceberg tables. The available values     ``HIVE``
+======================================================= ============================================================= ================================== =================== =========================================
+Property Name                                           Description                                                   Default                            Presto Java Support Presto C++ Support
+======================================================= ============================================================= ================================== =================== =========================================
+``iceberg.catalog.type``                                The catalog type for Iceberg tables. The available values     ``HIVE``                           Yes                 Yes, only needed on coordinator
                                                         are ``HIVE``, ``HADOOP``, and ``NESSIE`` and ``REST``.
 
-``iceberg.hadoop.config.resources``                     The path(s) for Hadoop configuration resources.
+``iceberg.hadoop.config.resources``                     The path(s) for Hadoop configuration resources.                                                  Yes                 Yes, only needed on coordinator
 
                                                         Example: ``/etc/hadoop/conf/core-site.xml.`` This property
                                                         is required if the iceberg.catalog.type is ``hadoop``.
                                                         Otherwise, it will be ignored.
 
-``iceberg.file-format``                                 The storage file format for Iceberg tables. The available     ``PARQUET``
+``iceberg.file-format``                                 The storage file format for Iceberg tables. The available     ``PARQUET``                        Yes                 NA
```

Comment:
Can we add a little explanation here why it's NA? We can say "NA, write is not supported yet"

Reply:
Same for other NA items. Thanks!